buildscripts/java-integration-test.gradle

```diff
@@ -89,3 +89,7 @@ tasks.register('jacocoIntegrationTestReport', JacocoReport) {
         html.required = true
     }
 }
+
+spotbugsIntegrationTest {
+    enabled(false)
+}
```

Comment:
A trailing newline is missing. We should probably have this automatically checked.